src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx

```diff
@@ -0,0 +1,52 @@
+darkthumbnail: ./kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking.webp
+eurl: "https://www.sdxcentral.com/articles/news/kubernetes-1-28-improves-open-source-cloud-native-compute-and-networking/2023/08/"
+published: true
+type: News
+---
+
+import { NewsWrapper } from "../../News.style.js";
+import { Link } from "gatsby";
+
+<NewsWrapper>
+
+The open-source Kubernetes cloud-native platform is out with its second major update of 2023, introducing a long list of enhancements for operators in the new 1.28 release.
+
+Kubernetes is an open-source project, originally started by Google and now developed under the Linux Foundation’s Cloud Native Computing Foundation (CNCF), benefiting from the contributions of more than 900 companies. Among the updates in Kubernetes 1.28 are enhancements designed to help with resiliency, including the ability to recover from non-graceful node shutdowns. There are also a series of networking-related enhancements that will enable better security and performance.
+
+As is the case with every Kubernetes release, there is a theme and a code name; for Kubernetes 1.28 the name is “Planternetes.” “So in the northern hemisphere, it’s summer right now and I think the symbolism of the garden and our community goes hand in hand,” Grace Nguyen, release lead for Kubernetes 1.28, told SDxCentral. “Each of us has a very important role in the ecosystem and together we built this big, open-source project that has a lot of impact.”
+
+<h3>Kubernetes 1.28 has deep roots for stability</h3>
+Nguyen noted that there are some 45 enhancements in the Kubernetes 1.28 update, spanning new stable, beta and alpha capabilities.
+
+Kubernetes is widely deployed on public cloud providers and is also used as the foundation of several commercial offerings, including one from IBM‘s Red Hat business unit.
+
+“The latest release of Kubernetes has a number of enhancements to help increase the stability, performance and maintainability of the core platform while also augmenting workload innovation, including AI [artificial intelligence] and virtual machines,” Karena Angell, OpenShift Commons lead and senior principal product manager for Red Hat OpenShift and Hybrid Platforms, told SDxCentral.
+
+Angell said Red Hat is particularly interested in the new node system memory swap support that has been added to Kubernetes 1.28 to enable better memory performance across a cluster. Additionally, she noted that the Kubernetes Job API now allows for more choices in AI model training and retraining, which is especially key for AI-driven and intelligent workloads.
+
+“As Kubernetes continues to mature as a platform with release 1.28, we’re very happy to see a number of new features and tweaks that further support production stability and consistency,” Angell said. “Large and complicated jobs can now fail faster and more accurately which, while it sounds strange, contributes to better overall performance of Kubernetes in production.”
```

Comment:
@Ghat0tkach Please move this section down, and bring the "Networking enhancements in Kubernetes 1.28" up.

Reply:
Yes doing the same